test/unit/create-contentful-api-test.js

```diff
@@ -352,3 +352,65 @@ test('CDA call sync fails', (t) => {
     teardown()
   })
 })
+
+test('Given json should be parsed correctly as a collection of entries', (t) => {
+  const client = contentful.createClient({accessToken: 'blablabla', space: 'bla'})
```

Comment:
That's completely different from what we are trying to achieve since we don't want to rewire the function we want the real logic to be executed